File: app/src/main/java/bassamalim/hidaya/database/AyatRecitersDB.java
package bassamalim.hidaya.database;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "ayat_reciters")
public class AyatRecitersDB {

    @PrimaryKey
    @ColumnInfo(name = "rec_id")
    private final int rec_id;
    @ColumnInfo(name = "rec_name")
    private final String rec_name;

    public AyatRecitersDB(int rec_id, String rec_name) {
        this.rec_id = rec_id;
        this.rec_name = rec_name;
    }

    public int getRec_id() {
        return rec_id;
    }

    public String getRec_name() {
        return rec_name;
    }
}

File: app/src/main/java/bassamalim/hidaya/core/data/database/dbs/AyatRecitersDB.kt
package bassamalim.hidaya.core.data.database.dbs

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "ayat_reciters")
data class AyatRecitersDB(
    @field:ColumnInfo(name = "rec_id") @field:PrimaryKey val reciterId: Int,
    @field:ColumnInfo(name = "rec_name") val reciterName: String?
)